internal/statestore/redis.go

```diff
@@ -672,24 +659,15 @@ func (rb *redisBackend) DeleteTicketsFromIgnoreList(ctx context.Context, ids []s
 	}
 	defer handleConnectionClose(&redisConn)
 
-	err = redisConn.Send("MULTI")
-	if err != nil {
-		redisLogger.WithError(err).Error("failed to pipeline commands for DeleteTicketsFromIgnoreList")
-		return status.Error(codes.Internal, err.Error())
+	cmds := make([]interface{}, len(ids)+1)
+	cmds[0] = "proposed_ticket_ids"
+	for i, id := range ids {
+		cmds[i+1] = id
```

Comment:
minor nit - append() over index manipulation? less bug prone for future changes?